### ISSUE TITLE
`java.lang.IllegalStateException` displayed in console when player joins server

### ISSUE THREAD

**Issue by BredyAK** [Updated-NoCheatPlus/NoCheatPlus#287]
#### *Complete* output of the `ncp version` command (please prefer text here)

```
> ncp version
[01:02:44 INFO]: »Version information«
[01:02:44 INFO]: Server:
[01:02:44 INFO]: git-Paper-446 ~MC: 1.19.3~
[01:02:44 INFO]: Detected: 1.19.3
[01:02:44 INFO]: NoCheatPlus:
[01:02:44 INFO]: Plugin: 3.17.1-SNAPSHOT-Updated-b154
[01:02:44 INFO]: MCAccess: 1.13-1.19|? / Bukkit-API
[01:02:44 INFO]: Features:
[01:02:44 INFO]: blocks: BlocksMC1_4, BlocksMC1_5, BlocksMC1_6_1, BlocksMC1_7_2, BlocksMC1_8, BlocksMC1_9, BlocksMC1_10, BlocksMC1_11, BlocksMC1_12, BlocksMC1_13, BlocksMC1_14, BlocksMC1_15, BlocksMC1_16, BlocksMC1_17, BlocksMC1_19, MCAccessBukkitModern
[01:02:44 INFO]: checks: FastConsume, Gutenberg, HotFixFallingBlockPortalEnter, AttackFrequency, FlyingFrequency, KeepAliveFrequency
[01:02:44 INFO]: defaults: pvpKnockBackVelocity
[01:02:44 INFO]: packet-listeners: UseEntityAdapter, MovingFlying, OutgoingPosition, KeepAliveAdapter, WrongTurnAdapter, NoSlow, Fight
[01:02:44 INFO]: Hooks:
AllViolations~NCP~ 1.0, ViolationFrequency~NCP~ 1.0
[01:02:44 INFO]: »Related Plugins«
[01:02:44 INFO]: ProtocolLib v5.0.0-SNAPSHOT-b614
```
#### Short description of the issue and how to reproduce (is it random / always happens / side conditions ?)

install the plugins above then start server, after player joins server, the warnings will display as below:
```
[22:27:39] [Server thread/INFO]: player_not_exist[/119.4.206.195:50833] logged in with entity id 126 at ([world]96.37382848718761, 66.0, -97.03904636581119)
[22:27:42] [Server thread/WARN]: java.lang.IllegalStateException: Unable to invoke method public static org.bukkit.Sound org.bukkit.craftbukkit.v1_19_R2.CraftSound.getBukkit(net.minecraft.sounds.SoundEffect)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.reflect.accessors.DefaultMethodAccessor.invoke(DefaultMethodAccessor.java:25)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.wrappers.BukkitConverters$20.getSpecific(BukkitConverters.java:1131)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.wrappers.BukkitConverters$20.getSpecific(BukkitConverters.java:1117)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.wrappers.Converters$1.getSpecific(Converters.java:50)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.wrappers.Converters$7.getSpecific(Converters.java:254)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.reflect.StructureModifier.readInternal(StructureModifier.java:301)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.reflect.StructureModifier.readSafely(StructureModifier.java:275)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.reflect.StructureModifier.getValues(StructureModifier.java:484)
[22:27:42] [Server thread/WARN]: 	at NoCheatPlus-b154.jar//fr.neatmonster.nocheatplus.checks.net.protocollib.SoundDistance.isSoundMonitoredLatest(SoundDistance.java:125)
[22:27:42] [Server thread/WARN]: 	at NoCheatPlus-b154.jar//fr.neatmonster.nocheatplus.checks.net.protocollib.SoundDistance.isSoundMonitored(SoundDistance.java:139)
[22:27:42] [Server thread/WARN]: 	at NoCheatPlus-b154.jar//fr.neatmonster.nocheatplus.checks.net.protocollib.SoundDistance.handleSoundPacket(SoundDistance.java:153)
[22:27:42] [Server thread/WARN]: 	at NoCheatPlus-b154.jar//fr.neatmonster.nocheatplus.checks.net.protocollib.SoundDistance.onPacketSending(SoundDistance.java:181)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.SortedPacketListenerList.invokeSendingListener(SortedPacketListenerList.java:195)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:149)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.PacketFilterManager.postPacketToListeners(PacketFilterManager.java:547)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.PacketFilterManager.invokePacketSending(PacketFilterManager.java:521)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.netty.manager.NetworkManagerInjector.onPacketSending(NetworkManagerInjector.java:99)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelInjector.processOutbound(NettyChannelInjector.java:571)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.netty.channel.NettyChannelInjector$2.doProxyRunnable(NettyChannelInjector.java:472)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.proxyRunnable(NettyEventLoopProxy.java:43)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.injector.netty.channel.NettyEventLoopProxy.execute(NettyEventLoopProxy.java:252)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.network.NetworkManager.sendPacket(NetworkManager.java:432)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.network.NetworkManager.processQueue(NetworkManager.java:547)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.network.NetworkManager.flushQueue(NetworkManager.java:503)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.network.NetworkManager.a(NetworkManager.java:398)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizen.nms.v1_19.impl.network.handlers.DenizenNetworkManagerImpl.a(DenizenNetworkManagerImpl.java:324)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2157)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2143)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizen.nms.v1_19.impl.network.handlers.AbstractListenerPlayInImpl.a(AbstractListenerPlayInImpl.java:119)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizen.nms.v1_19.impl.network.handlers.DenizenPacketListenerImpl.a(DenizenPacketListenerImpl.java:126)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.players.PlayerList.a(PlayerList.java:1176)
[22:27:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R2.CraftWorld.playSound(CraftWorld.java:1843)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizen.scripts.commands.world.PlaySoundCommand.execute(PlaySoundCommand.java:131)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.scripts.commands.CommandExecutor.execute(CommandExecutor.java:114)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.scripts.queues.ScriptEngine.revolve(ScriptEngine.java:56)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.scripts.queues.core.InstantQueue.onStart(InstantQueue.java:19)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.scripts.queues.ScriptQueue.start(ScriptQueue.java:338)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.events.ScriptEvent.run(ScriptEvent.java:803)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.events.ScriptEvent.lambda$fire$2(ScriptEvent.java:771)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.utilities.scheduling.OneTimeSchedulable.tick(OneTimeSchedulable.java:17)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.DenizenCore.tick(DenizenCore.java:306)
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizen.Denizen.lambda$onEnable$1(Denizen.java:416)
[22:27:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftTask.run(CraftTask.java:101)
[22:27:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1473)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:447)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1397)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1173)
[22:27:42] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316)
[22:27:42] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
[22:27:42] [Server thread/WARN]: Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.resources.MinecraftKey.b()" because "minecraft" is null
[22:27:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R2.util.CraftNamespacedKey.fromMinecraft(CraftNamespacedKey.java:24)
[22:27:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_19_R2.CraftSound.getBukkit(CraftSound.java:28)
[22:27:42] [Server thread/WARN]: 	at ProtocolLib-b614.jar//com.comphenix.protocol.reflect.accessors.DefaultMethodAccessor.invoke(DefaultMethodAccessor.java:23)
[22:27:42] [Server thread/WARN]: 	... 49 more
[22:27:42] [Craft Scheduler Thread - 593 - NoCheatPlus/INFO]: [NoCheatPlus] Unregistered packet level hook:fr.neatmonster.nocheatplus.checks.net.protocollib.SoundDistance
```
#### Have you made any (noteworthy) changes to the default configuration of NCP?

nope.
#### Do you run any other special plugins that affect game mechanics next to NC+ (skills, machines, adding/changing blocks, other anti-cheating)?

nope.
#### Provide additional information: for bypasses, vulnerabilities and reproducible issues, we request a [debug log. ](https://github.com/Updated-NoCheatPlus/Docs/blob/master/Development/Debugging.md#on-the-fly-debug-output-for-individual-players) Use [GitHub gists](https://gist.github.com/) or [PasteBin](http://pastebin.com/) or similar for errors and huge logs, or attach a (zip-) file here.

nope.

**Comment by F3F5** [comment]
p.s protocollib buggy in dev build(last version) in 24.03 (WARNING) not working on 1.12.2 -  1.8 in my test (i created issues)
USE B14 PROTOCOLLIB

**Comment by ghost** [comment]
[22:27:42] [Server thread/WARN]: 	at Denizen-1.2.6-b6698-DEV.jar//com.denizenscript.denizencore.scripts.queues.ScriptEngine.revolve(ScriptEngine.java:56)

**Comment by Lysandr0** [comment]
Fixed by PLib, issue #2276
